unicode-data.cabal

```diff
@@ -110,10 +112,43 @@ executable ucd2haskell
   if flag(ucd2haskell)
     buildable: True
     build-depends:
-        base             >= 4     && < 5
+        base             >= 4.7   && < 5
       , streamly         >= 0.8   && < 0.9
       , getopt-generics  >= 0.13  && < 0.14
       , containers       >= 0.5   && < 0.7
       , directory        >= 1.3.6 && < 1.3.7
   else
     buildable: False
+
+test-suite test
+  import: default-extensions, compile-options
+  type: exitcode-stdio-1.0
+  main-is: Main.hs
+  hs-source-dirs:
+      test
+  other-modules:
+      Unicode.CharSpec
+  build-depends:
+      base             >= 4.7   && < 5
+    , hspec            >= 2.0   && < 3
+    , unicode-data
+  build-tool-depends:
+      hspec-discover:hspec-discover >= 2.0 && < 3
+  -- We need to match a GHC version with the same Unicode version.
+  -- See: test/Unicode/CharSpec.hs for compatibility table.
+  if impl(ghc >= 9.2.1)
+    cpp-options: -DCOMPATIBLE_GHC_UNICODE
+  default-language: Haskell2010
+
+benchmark Unicode.Char
+  import: default-extensions, compile-options
+  type: exitcode-stdio-1.0
+  hs-source-dirs: bench
+  main-is: Main.hs
+  build-depends:   
```

Comment:
Remove additional whitespace at the end of the line.

Reply:
Fixed